src/drivers/distance_sensor/lightware_sf45_serial/module.yaml

```diff
@@ -41,11 +41,11 @@ parameters:
                       The SF45 mounted facing upward or downward on the frame
               type: enum
               values:
-                0: Rotation upward
-                1: Rotation downward
+                24: Rotation upward
+                25: Rotation downward
```

Comment:
I never actually had the chance to test these on the bench. The non-standard Yaw cfg as well.

Reply:
just tested them all, and they still work as intended

Reply:
Nice!